tests/documentation/DocumentationStyleBearTest.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from bears.documentation.DocumentationStyleBear import DocumentationStyleBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+def load_testfile(test, filename):
+    filepath = os.path.join(os.path.dirname(__file__), "test_files",
+                            "DocumentationStyleBear", filename)
+    with open(filepath) as fl:
+        return fl.read()
+
+good_file = load_testfile("DocumentationStyleBear", "good_file.py")
+bad_file = load_testfile("DocumentationStyleBear", "bad_file.py")
+
+file_sameline = load_testfile("DocumentationStyleBear", "bad_file2.py")
```

Comment:
Why declare them in different files ? Why not put them in strings here ?
Adding strings here makes it easier to find and read rather than searching for test_files and switching between editor tabs, etc.